packages/mobile/src/verify/VerificationEducationScreen.tsx

```diff
@@ -142,6 +143,13 @@ function VerificationEducationScreen({ route, navigation }: Props) {
     dispatch(checkIfKomenciAvailable())
   }, [])
 
+  // CB TEMPORARY HOTFIX: Pinging Komenci endpoint to ensure availability
+  const hideVerification = useSelector(hideVerificationSelector)
+  const asyncKomenciAvailable = useAsync<boolean>(async () => {
+    const response = await fetch(networkConfig.komenciLoadCheckEndpoint)
+    return response.json()
```

Comment:
ditto has above. See Jean's PR for an alternative to this version (sorry if you know that already)